src/utils/sourcemaps.rs

```diff
@@ -162,18 +178,36 @@ fn guess_sourcemap_reference(sourcemaps: &HashSet<String>, min_url: &str) -> Res
             parts[parts_len - 1] = map_ext;
             let new_ext = parts.join(".");
             if sourcemaps.contains(&unsplit_url(path, basename, Some(&new_ext))) {
-                return Ok(unsplit_url(None, basename, Some(&new_ext)));
+                return Ok(SourceMapReference::from_url(unsplit_url(
+                    None,
+                    basename,
+                    Some(&new_ext),
+                )));
             }
         }
     }
 
     bail!("Could not auto-detect referenced sourcemap for {}", min_url);
 }
 
+pub struct SourceMapReference {
```

Comment:
Please add some documentation to this struct, it's not easy to understand what `url` and `original_url` are.